src/app/components/autocomplete/autocomplete.ts

```diff
@@ -1245,7 +1245,16 @@ export class AutoComplete implements AfterViewChecked, AfterContentInit, OnDestr
 
     onArrowDownKey(event) {
         if (!this.overlayVisible) {
-            return;
+            if(this.focused) {
```

Comment:
IMO It is more clear and escape of the function fast if not `this.overlayVisible || !this.focused`.
Try it if continue works if you want :)IMO It is more clear and escape of the function fast if not `this.overlayVisible || !this.focused`.
Try it if  works,if you want :)

onArrowDownKey(event) {
    if (this.overlayVisible || !this.focused) {
        return;
    }

    const el = this.inputEL.nativeElement as HTMLInputElement | undefined;

    if (el) {
        // Put cursor at the end as the preventDefault blocks this behaviour
        el.setSelectionRange(el.value.length, el.value.length);
    }

    this.overlayVisible = true;
}

Reply:
Your suggestion has omitted some existing code, which should run when this.overlayVisible is true, so I don't think it would work as expected